src/components/TodoList/TodoList.jsx

```diff
@@ -1,44 +1,114 @@
 import React from 'react';
 import './TodoList.scss';
+import classnames from 'classnames';
+import PropTypes from 'prop-types';
+import { getTodos } from '../../api';
 
-export const TodoList = () => (
-  <div className="TodoList">
-    <h2>Todos:</h2>
-
-    <div className="TodoList__list-container">
-      <ul className="TodoList__list">
-        <li className="TodoList__item TodoList__item--unchecked">
-          <label>
-            <input type="checkbox" readOnly />
-            <p>delectus aut autem</p>
-          </label>
-
-          <button
-            className="
-              TodoList__user-button
-              TodoList__user-button--selected
-              button
-            "
-            type="button"
-          >
-            User&nbsp;#1
-          </button>
-        </li>
-
-        <li className="TodoList__item TodoList__item--checked">
-          <label>
-            <input type="checkbox" checked readOnly />
-            <p>distinctio vitae autem nihil ut molestias quo</p>
-          </label>
-
-          <button
-            className="TodoList__user-button button"
-            type="button"
-          >
-            User&nbsp;#2
-          </button>
-        </li>
-      </ul>
-    </div>
-  </div>
-);
+export class TodoList extends React.Component {
+  state = {
+    title: '',
+    visibleTodos: [],
+    status: 0,
+  }
+
+  async componentDidMount() {
+    const todos = await getTodos();
+
+    this.setState({
+      visibleTodos: [...todos],
+    });
+  }
+
+  render() {
+    const { selectUser } = this.props;
+    let { visibleTodos } = this.state;
+    const { status, title } = this.state;
+
+    if (status !== 0) {
+      visibleTodos = (status === 1)
```

Comment:
Don't mutate the state.
Use `setState` instead, e.g:
`setState({ visibleTodos: (prevState) => visibleTodos: prevState.visibleTodos.filter(el => el.completed)) }` etc